website/src/components/canvas/sidebar/ResizeTextfields.jsx

```diff
@@ -104,37 +143,43 @@ class ResizeTextfields extends React.Component {
                   iconStyle={styles.smallIcon}
                   style={styles.small}
                   onTouchTap={this.generateButtonHandler(-1 * ONE_UNIT, HEIGHT)}
+                  disabled={this.state.disableTextfields}
                 >
                     <RemoveCircle/>
                 </IconButton>
                 <TextField
                   onChange={this.handlerTextfieldHeight}
                   floatingLabelText="Height"
                   style={styles.field}
+                  disabled={this.state.disableTextfields}
                 />
                 <IconButton
                   iconStyle={styles.smallIcon}
                   style={styles.small}
                   onTouchTap={this.generateButtonHandler(ONE_UNIT, HEIGHT)}
+                  disabled={this.state.disableTextfields}
```

Comment:
Just a suggestion: I would change the styling so that if something is disabled that like warning cursor thing doesn't appear.